dd-sdk-android/src/main/kotlin/com/datadog/android/rum/RumMonitor.kt

```diff
@@ -170,6 +171,15 @@ interface RumMonitor {
         attributes: Map<String, Any?>
     )
 
+    /**
+     * Adds a specific timing in the active View. The timing duration will be computed as the
+     * difference between the time the View was started and the time this function was called.
+     * @param name the name of the new custom timing attribute
```

Comment:
How about adding a note on supported / illegal characters in the timing `name`?

Reply:
In that case we will have to do so also for the other attributes (Logger, RUM). Not sure what to say.